src/App.js

```diff
@@ -1,20 +1,21 @@
-
 import './App.css';
 import Header from './components/Header';
 import "bootstrap/dist/css/bootstrap.min.css";
 import "bootstrap/dist/js/bootstrap.bundle.min";
 import PackageCard from './components/PackageCard';
+import SignInUpPage from './components/SignInUpPage';
+import {Routes, Route} from 'react-router-dom';
+
 
 function App() {
   return (
     <div className="App">
       <Header />
-      <PackageCard
```

Comment:
Could you please put this code back? just want to leave the template until we implement the database!

Reply:
Ok, will do!